src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,80 @@
 package core.basesyntax;
 
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvValidationException;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int FIRST_POINT = 0;
+    private static final int SECOND_POINT = 1;
+    private static final String SUPPLY_LINE = "supply";
+    private static final String BUY_LINE = "buy";
+    private static final String RESULT_LINE = "result";
+    private static final String COMA = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String value = getDataFromFile(fromFileName);
+        String sentence = dataCalculation(value);
+        writeToFile(sentence, toFileName);
+    }
+
+    private String getDataFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (CSVReader csvReader = new CSVReader(new FileReader(fromFileName))) {
+            String[] nextLine;
+            while (true) {
+                try {
+                    if ((nextLine = csvReader.readNext()) == null) {
+                        break;
+                    }
+                } catch (IOException | CsvValidationException e) {
+                    throw new RuntimeException(e);
+                }
+                stringBuilder.append(nextLine[FIRST_POINT])
+                        .append(",")
+                        .append(nextLine[SECOND_POINT])
+                        .append(" ");
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file", e);
+        }
+        return stringBuilder.toString().trim();
+    }
+
+    private String dataCalculation(String value) {
```

Comment:
```suggestion
    private String calculateData(String value) {
```
 method names should be verbs